### DISCUSSION TITLE
Bridging Support?

### DISCUSSION THREAD

**Discussion by schmidtw** [nanomq/nanomq#159] [Q&A] [answered]
Hi, nanomq looks interesting.  Does it support bridging?

**Comment by JaylinYu** [comment]
Hi, bridging will be supported on Ver 0.5.2

**Comment by JaylinYu** [comment, marked as answer]
Hi bro, in case your are still waiting. Now NanoMQ supports MQTT bridging in the updated master branch. I will issue a release after performance tunning.
